### ISSUE TITLE
Image upload on mobile

### ISSUE THREAD

**Issue by gignar** [prose/prose#828]
I feel like I'm missing something obvious, but how are you supposed to upload images on mobile? The only references to image uploading I can find all specify drag-n-drop into the editor, which obviously isn't possible on a phone.

**Comment by Koxzi95** [comment]
It doesn't look like you can. I don't think prose was really built with mobile editing in mind. Considering there's no toolbar.

**Comment by dereklieu** [comment]
There's no way to do this right now. Creating a mobile label to categorize these issues.

**Comment by phillipadsmith** [comment]
> There's no way to do this right now. Creating a mobile label to categorize these issues.

@dereklieu: Is this specifically a Codemirror issue? Mostly just curious...

**Comment by dereklieu** [comment]
@phillipadsmith No, nothing to do with Codemirror. The toolbar where the image upload dialog would usually be is hidden on mobile, so at the very least this would be some UI and styling work.

**Comment by ndubbaka** [comment]
looks like the toolbar is being hidden at device width less than or equal to 480px.

```
@media only screen and (min-device-width: 320px) and (max-device-width: 480px)
.details .mask, .edit .topbar-wrapper, .popup {
    display: none !important;
}
```
